docs/topics/extensions.rst

```diff
@@ -66,7 +66,7 @@ Disabling an extension
 
 In order to disable an extension that comes enabled by default (ie. those
 included in the :setting:`EXTENSIONS_BASE` setting) you must set its order to
-``None``. For example::
+``None``(or ``0``). For example::
```

Comment:
Oh, I see. In that example, the `MYEXT_ENABLED` setting is a boolean (like many similar builtin settings, e.g. `LOG_ENABLED` or `DNSCACHE_ENABLED`). `EXTENSIONS` on the other hand is quite different, in that it is a dictionary.